### ISSUE TITLE
VSCode UI problem when using Remote Desktop

### ISSUE THREAD

**Issue by kuhlaid** [microsoft/vscode#141220]
Issue Type: <b>Bug</b>

This might be more a problem with remote desktop for Window, but here goes. I am accessing a Window 10 computer via Remote Desktop. The remote machine is running VSCode with a NodeJs project. If I start the node dev server and then exit out of the Remote Desktop connection (leaving the remote machine running), when I log back into the Remote Desktop connection to go back into VSCode, I can't work with the UI. UI refresh/redraw is completely out of whack and I have to go into the Windows task bar or task manager to close VSCode because the minimize and close buttons at the top right of VSCode stop functioning.

Below is a screen cast of VSCode before I disconnect from Remote Desktop.

When I reconnect to Remote Desktop this is a screen cast of how the UI is broken. The main menus seem to stick to the window when selecting them, when scrolling the code window the right portion of the window scrolls but leaves the code lines on the left in place and you can see the visible break in the code window as I scroll.

VS Code version: Code 1.63.2 (899d46d82c4c95423fb7e10e68eba52050e30ba3, 2021-12-15T09:40:02.816Z)
OS version: Windows_NT x64 10.0.19043
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-10600 CPU @ 3.30GHz (12 x 3312)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.71GB (3.47GB free)|
|Process Argv|--crash-reporter-id dfcb96df-14f2-437a-8f49-b9719d1003cc|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (17)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-markdownlint|Dav|0.45.0
vscode-eslint|dba|2.2.2
python-environment-manager|don|1.0.3
python-extension-pack|don|1.7.0
rest-client|hum|0.24.6
vsc-python-indent|Kev|1.14.2
MagicPython|mag|1.1.0
rainbow-csv|mec|2.0.0
dotenv|mik|1.0.1
python|ms-|2021.12.1559732655
vscode-pylance|ms-|2022.1.3
remote-wsl|ms-|0.63.13
vscode-typescript-next|ms-|4.6.20220121
autodocstring|njp|0.5.4
svelte-vscode|sve|105.10.0
vscodeintellicode|Vis|1.2.15
markdown-all-in-one|yzh|3.4.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyl392:30425749
pythontb:30283811
pythonvspyt551cf:30345471
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscoreces:30384385
pythondataviewer:30285071
vscod805cf:30301675
pythonvspyt200:30340761
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593:30376534
vsc1dst:30424896
pythonvs932:30410667
wslgetstartedc:30426334
vscop804:30404766
vscop453:30404998
vsclayoutctrt:30425946
vsrem710:30416614
vsbas813:30426126
vsic-wlc-tsjs:30426436

```

</details>

**Comment by TheBrenny** [comment]
This should be closed in favour of #141221 because it actually has the screencasts and the same body of text (minus the Sysinfo, Extensions and A/B blocks.

**Comment by deepak1556** [comment]
Merging to https://github.com/microsoft/vscode/issues/167556
